jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/definition/list/ProcessDefinitionListViewImpl.java

```diff
@@ -228,9 +239,9 @@ public ProcessSummary getValue( ProcessSummary object ) {
     }
 
     public void refreshNewProcessInstance( @Observes NewProcessInstanceEvent newProcessInstance ) {
-        PlaceStatus definitionDetailsStatus = placeManager.getStatus( new DefaultPlaceRequest( "Process Details Multi" ) );
+        PlaceStatus definitionDetailsStatus = placeManager.getStatus( new DefaultPlaceRequest( placeIdentifier ) );
```

Comment:
This one is throwing a NPE the first time that we are starting a new Process Instance, because probably the placeIdentifier field is not initialised yet